src/CMakeLists.txt

```diff
@@ -353,6 +353,8 @@ if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   target_link_libraries(openroad mpl2 par)
   target_compile_definitions(openroad PRIVATE ENABLE_MPL2)
   target_compile_definitions(openroad PRIVATE ENABLE_PAR)
+  add_subdirectory(mpl2)
+  add_subdirectory(par)
```

Comment:
Do the ENABLE_* no longer work?

Reply:
Cherry mentioned that just adding the subdir was causing problems and suggested this patch.

Reply:
ok then we might as well drop the options

Reply:
mpl2 and par compile and link just fine on macOS M1.

The resulting openroad executable will throw a runtime error. 
Putting the ortools.dylib at the end of the link command fixes that.

Reply:
the `ENABLE_PAR` and `ENABLE_MPL2` are required for correct includes in `Main.cc` and `OpenRoad.cc`